lib/internal/Magento/Framework/Config/Dom.php

```diff
@@ -380,7 +381,7 @@ public static function validateDomDocument(
         try {
             $result = $dom->schemaValidate($schema);
             if (!$result) {
-                $errors = self::getXmlErrors($errorFormat);
+                $errors = self::getXmlErrors($errorFormat, $dom);
```

Comment:
Can we pass the `$dom` object in all the places where we're calling the `getXmlErrors` method?

Reply:
was unable to replicate the scenario where `!$result` and didn't want to add somewhere I couldn't test.
But yes, I can't see arason why not.